src/content/events/2024/hackathon-march-2024/germany-heidelberg.md

```diff
@@ -104,11 +108,16 @@ This local event is being kindly supported by the German Human Genome-Phenome Ar
 
 <div style="text-align: center;">
     <img src="https://www.ghga.de/fileadmin/user_upload/Bilder/Logo/GHGA_CMYK_Bright_Orange.svg" width=300>
-    <img src="https://www.nct-heidelberg.de/fileadmin/sites/nct-heidelberg.de/website/img/nct-heidelberg.jpg" width=300>
+    <img src="https://www.ghga.de/fileadmin/user_upload/Bilder/Logo/NCT-logo.png" width=300>
+</div>
+
+<div style="text-align: center;">
+    <img src="https://www.ghga.de/fileadmin/user_upload/Bilder/Logo/Master-Logo.png" width=250>
+    <img src="https://www.ghga.de/fileadmin/user_upload/Bilder/Logo/dktk-logo.png" width=400>
```

Comment:
any higher resultion version of the last image available? Also do you maybe have a version for dark mode? I can help out on how to add them, if you can provide links.

Reply:
I will ask relevant people because i dont have the logos (they sent me these low-quality ones :D ). We can keep the link the same for the new logos though.

Reply:
@kubranarci better?

Reply:
thanks, @heylf seems enough for now :) I think we can merge it for now, we can change the logos later without playing with the links.